pwa/src/templates/templateParts/dependenciesTemplates/networkDependenciesTemplate/NetworkDependenciesTemplate.module.css

```diff
@@ -0,0 +1,24 @@
+:export {
+  layerColorInteractie: --web-app-layer-color-interaction;
+  layerColorProces: --web-app-layer-color-process;
+  layerColorIntegratie: --web-app-layer-color-integration;
+  layerColorService: --web-app-layer-color-services;
+  layerColorData: --web-app-layer-color-data;
+  borderLayerColorInteractie: --web-app-border-layer-color-interaction;
+  borderLayerColorProces: --web-app-border-layer-color-process;
+  borderLayerColorIntegratie: --web-app-border-layer-color-integration;
+  borderLayerColorService: --web-app-border-layer-color-services;
+  borderLayerColorData: --web-app-border-layer-color-data;
+  highlightLayerColorInteractie: --web-app-layer-category-color-interaction;
+  highlightLayerColorProces: --web-app-layer-category-color-process;
+  highlightLayerColorIntegratie: --web-app-layer-category-color-integration;
+  highlightLayerColorService: --web-app-layer-category-color-services;
+  highlightLayerColorData: --web-app-layer-category-color-data;
```

Comment:
I suggest to remove these highlight colors for the same reason as the borders. However, to keep a "highlighted" state, we could use opacity instead.

So, without a token, we could just set the opacity of the balls background. When highlighted.

Reply:
Note: I think it's even debatable to remove the highlighted state altogether since it doesn't do anything (you're not getting extra content or any other functionality out of it).